runhouse/resources/hardware/utils.py

```diff
@@ -410,15 +407,28 @@ def parse_filters(since: str, cluster_status: str):
 def get_clusters_from_den(cluster_filters: dict):
     get_clusters_params = {"resource_type": "cluster", "folder": rns_client.username}
 
+    if "cluster_status" in cluster_filters:
+        # Include the relevant daemon status for the filter based on the cluster status provided
+        if cluster_filters["cluster_status"] == ClusterStatus.RUNNING:
+            cluster_filters["status"] = RunhouseDaemonStatus.RUNNING
```

Comment:
daemon doesnt necessarily need to be running if the cluster is up right?

Reply:
that's true, this is a little tricky tho - bc this is meant to support clusters (at least for now) not launched via the launcher where we only have the daemon status - but given we're moving towards all launching happening via the launcher we can prob remove and rely just on the cluster status